_config.yml

```
@@ -19,7 +19,7 @@ lang: en # the language of your site (for example: en, fr, cn, ru, etc.)
 icon: ⚛️  # the emoji used as the favicon (alternatively, provide image name in /assets/img/)
 
 url: https://alshedivat.github.io # the base hostname & protocol for your site
-baseurl: /al-folio # the subpath of your site, e.g. /blog/
+# baseurl: /al-folio # the subpath of your site, e.g. /blog/
```

I have removed `baseurl` since it is causing some problems while deploying this branch to Vercel. I will revert this change before merging this PR.

---

```suggestion
baseurl: /al-folio # the subpath of your site, e.g. /blog/
```
Has to be done before merging.